build/git_revision_test.go

```diff
@@ -35,12 +42,15 @@ func TestGitRevision_terraform(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	licensePath := filepath.Join(filepath.Dir(execPath), "LICENSE.txt")
 	t.Cleanup(func() {
 		gr.Remove(ctx)
 		// check if license was deleted
-		if _, err := os.Stat(licensePath); !os.IsNotExist(err) {
-			t.Fatalf("license file not deleted at %q: %s", licensePath, err)
+		files, err := os.ReadDir(tempDir)
+		if err != nil {
+			t.Fatal("failed to read license directory in cleanup", err)
+		}
+		if len(files) != 0 {
+			t.Fatalf("license file not deleted, tempDir %q is not empty", tempDir)
```

Comment:
Is there a particular reason behind this change?

Reply:
It's because this PR adds a change to give any found license file the same name - before, any license file found (code searches for "LICENSE" and "LICENSE.txt" and copied over was always names "LICENSE.txt"
```
	// copy license file on best effort basis
	dstLicensePath := filepath.Join(installDir, "LICENSE.txt")
	srcLicensePath := filepath.Join(repoDir, "LICENSE.txt")
	altSrcLicensePath := filepath.Join(repoDir, "LICENSE")
```

But this change means the test no longer knows to expect "LICENSE.txt" as before:
```
licensePath := filepath.Join(filepath.Dir(execPath), "LICENSE.txt")
if _, err := os.Stat(licensePath); !os.IsNotExist(err) {
...
```

Instead, the test only knows the dir for the licenses. It could either construct paths for `<licensedir>/LICENSE` and <licenseir>/LICENSE.txt` and `os.Stat` those, but I think the easier approach was to check for number of files in the dir.

As we know the license dir must exist, I changed the test to check for files *within* the dir. Any license files should have been "tracked" and removed as part of the cleanup, so number of files within the license dir should be 0.

If you think there is a different approach around this, or if the keeping-the-filename-change isn't required (i.e. if it's alright to keep the found license file as the .txt version), just let me know 🙂

Reply:
See my comment above - I didn't notice the destination filename change in my previous review and didn't realise this was related. I'm assuming if we always copy it over as `LICENSE.txt` then no change is necessary here.